code/game/alternate_appearance.dm

```diff
@@ -136,6 +136,11 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 		return TRUE
 	return FALSE
 
+/datum/atom_hud/alternate_appearance/basic/heretic
+
```

Comment:
It's be very cool if you set `add_ghost_version = TRUE` to this hud so ghosts could see influences